content/en/agent/cluster_agent/clusterchecks.md

```diff
@@ -66,30 +87,28 @@ Enable the `clusterchecks` configuration provider on the Datadog **Node** Agent.
           polling: true
     ```
 
-[Restart the Agent][1] to apply the configuration change.
+[1]: /agent/cluster_agent/setup/
```

Comment:
Did you mean to delete the "Restart the Agent" line? if so, you can also remove the link URL (line 90)

Reply:
Yes meant to remove that, that URL link is also referenced earlier too on line 65 so kept it for that.